build.gradle

```diff
@@ -427,7 +427,8 @@ dependencyManagement {
       entry 'spring-messaging'
     }
 
-    dependencySet(group: 'org.yaml', version: '1.31') {
+    // CVE-2022-38752
+    dependencySet(group: 'org.yaml', version: '1.32') {
```

Comment:
I wen't to 1.33 on tribs can we go to that instead. This product gets a new vul like every week so would prefer we were right at the top.

Reply:
Agreed.